Solutions/CognyteLuminar/SolutionMetadata.json

```diff
@@ -0,0 +1,23 @@
+{
+    "publisherId":"Cognytetechnologiesltd",
+    "offerId":"",
```

Comment:
Hi @RamboV, you need to provide offer ID.
Refer - https://github.com/Azure/Azure-Sentinel/tree/master/Solutions#author-and-publish-a-solution-offer

Reply:
Hi @v-rbajaj , OfferID will be available where? because our client did the partnership, so just wanted little information on where they will find the offerId

Reply:
Hi @RamboV, please take a look at the below screenshot which is from the above shared link
![image](https://github.com/Azure/Azure-Sentinel/assets/120547590/9b24e088-7db4-4cfa-b753-c3535f7c0879)
I hope that helps..